<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetAtomWidget</class>
 <widget class="QWidget" name="SetAtomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>96</width>
    <height>122</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="4">
    <widget class="schrodinger::sketcher::ModularElementButton" name="last_picked_element_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
color: #333333;
}</string>
     </property>
     <property name="text">
      <string>Si</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">atom_group</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="cl_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Chlorine</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
color: #333333;
}</string>
     </property>
     <property name="text">
      <string>Cl</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">atom_group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="n_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Nitrogen</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
color: #333333;
}</string>
     </property>
     <property name="text">
      <string>N</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">atom_group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="c_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Carbon</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
color: #333333;
}</string>
     </property>
     <property name="text">
      <string>C</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">atom_group</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="s_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Sulphur</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
color: #333333;
}</string>
     </property>
     <property name="text">
      <string>S</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">atom_group</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="o_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Oxygen</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
color: #333333;
}</string>
     </property>
     <property name="text">
      <string>O</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">atom_group</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="p_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Phosphorus</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
color: #333333;
}</string>
     </property>
     <property name="text">
      <string>P</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">atom_group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="h_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Hydrogen</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
color: #333333;
}</string>
     </property>
     <property name="text">
      <string>H</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">atom_group</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="f_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Fluorine</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
color: #333333;
}</string>
     </property>
     <property name="text">
      <string>F</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">atom_group</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="schrodinger::sketcher::ModularToolButton" name="atom_query_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold italic 15px;
color: #606060;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">atom_group</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="schrodinger::sketcher::ToolButtonWithPopup" name="periodic_table_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Periodic Table</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../sketcher.qrc">
       <normaloff>:/icons/periodic_table.png</normaloff>:/icons/periodic_table.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>schrodinger::sketcher::ModularToolButton</class>
   <extends>QToolButton</extends>
   <header>schrodinger/sketcher/modular_tool_button.h</header>
  </customwidget>
  <customwidget>
   <class>schrodinger::sketcher::ToolButtonWithPopup</class>
   <extends>QToolButton</extends>
   <header>schrodinger/sketcher/tool_button_with_popup.h</header>
  </customwidget>
  <customwidget>
   <class>schrodinger::sketcher::ModularElementButton</class>
   <extends>QToolButton</extends>
   <header>schrodinger/sketcher/modular_element_button.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../sketcher.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="atom_group"/>
 </buttongroups>
</ui>
